### ISSUE TITLE
Find usages of variables.

### ISSUE THREAD

**Issue by JoshVarty** [CodeConnect/SourceBrowser#108]
I desperately need this feature. It's a colossal pain in the ass to use SourceBrowser without it.

When you click on the definition of a variable, its usages should show up in the search results. This is the behavior Reference Source has and it works really well even though you can't directly search for usages.

For example: http://source.roslyn.io/#Microsoft.CodeAnalysis/Syntax/SyntaxTree.cs,8649488200d5b57a,references

**Comment by AmadeusW** [comment]
Absolutely! I also really need this one. I'll do this one as I might do a podcast about doing this kind of tool.
I see that this is invoked by clicking on an identifier definition, (a link that takes you to exactly same place that you clicked) is this correct?

**Comment by ddurschlag** [comment]
Agreed. Have a separate issue for this on mu fork. I know how to implement it, it just takes time.
